frontend/src/scenes/insights/insightLogic.ts

```diff
@@ -668,6 +668,15 @@ export const insightLogic = kea<insightLogicType>({
                 )
             },
         ],
+
+        showTotalCount: [
```

Comment:
Shouldn't this live within insightsTableLogic? :thinking: 

As-is it's not relevant for all insights but rather trends/where we show the table.

Reply:
done